### PR TITLE
Correction of french corpus and config

### PR DESCRIPTION
**Pull request by missuniverse**
Corrections on the french text and sentences in both the corpus and the configs.
@mikehenrty 

Related to #1108

### CODE REVIEW

**Comment by mikehenrty** [comment]
@lissyx, would you mind have a look at this?

**Comment by lissyx** [comment]
I'm not sold on the "de" to "de la", that feels weird when reading. the diff of breton is waaay to big to be manually reviewed on github, it looks like it's completely different file :(

**Comment by missuniverse** [comment]
@lissyx Concerning the use of feminine for the country names, this is a dictionary reference:
http://www.larousse.fr/encyclopedie/departement/Guadeloupe_971/122457
https://www.larousse.fr/dictionnaires/francais-espagnol/Guadeloupe/100852
According to the dictionary, we should be using "de la" instead of "de".

Also, for the second file, there was over 500 duplicates and I had to change the order of the text to remove the duplicates, which is why it looks so different.

**Comment by lissyx** [comment]
Looking at each diff, even the one removing duplicates add a lot of new strings, it's hard to check.

**Comment by lissyx** [comment]
@missuniverse Your links to larousse does not give anything, while for instance I find https://fr.wikipedia.org/wiki/Fran%C3%A7ais_de_C%C3%B4te_d%27Ivoire

**Comment by lissyx** [comment]
@TheoChevalier any opinion there ?

**Comment by TheoChevalier** [comment]
Omitting the pronoun doesn’t make the noun masculine, so I disagree with this change and I suggest we keep the current translations.
We say `Je parle le français de France`, not `Je parle le français de la France`.

**Comment by missuniverse** [comment]
@lissyx 
I'll just say this as a comment, irony:
image

I guess it depends on where your from that it sounds right or not.

**Comment by lissyx** [comment]
Yes, please, and can you also make the other changes more readable ?

**Comment by missuniverse** [comment]
@mikehenrty @lissyx I removed the config file changes, but concerning making the changes more readable, the only way would be if someone sorted the original file, then the diff would be way easier to understand. But otherwise I don't know how I can make it more readable.

**Comment by lissyx** [comment]
@missuniverse you just gave the solution yourself, rebase and sort it before doing your changes :-)

**Comment by missuniverse** [comment]
@lissyx I adjusted the pull request like you said.

File: server/data/fr/ofis_publik_ar_brezhoneg.txt [hunk 1]
```diff
@@ -12,200 +11,180 @@
 Absolument identique.
 Absolument nulle part.
 À cause de cela.
-À cause de quoi ?
+À cause de quoi?
```
Comment (lissyx) [inline server/data/fr/ofis_publik_ar_brezhoneg.txt]: This is wrong.

File: server/data/fr/ofis_publik_ar_brezhoneg.txt [hunk 2]
```diff
@@ -12,200 +11,180 @@
...
 à ce propos.
 À certains endroits.
 À certains endroits il y a des manques.
 À ce soir.
-à ce sujet.
-à cette époque, les gens n'avaient pas beacoup de temps libre.
+À ce sujet.
```
Comment (lissyx) [inline server/data/fr/ofis_publik_ar_brezhoneg.txt]: Wrong as well

File: server/data/fr/ofis_publik_ar_brezhoneg.txt [hunk 3]
```diff
@@ -12,200 +11,180 @@
...
 À gauche.
 Ah, bon.
-Ah, c'est bon !
-Ah ! Tu es étais là ?
-à huit heures je suis rentré à la maison.
-aimes.
-aimez.
-aimons.
+Ah, c'est bon!
+Ah! Tu es étais là?
```
Comment (lissyx) [inline server/data/fr/ofis_publik_ar_brezhoneg.txt]: Wrong and wrong, you remove spaces

**Comment by missuniverse** [comment]
@lissyx To my knowledge there is no accepted use of a space before a punctuation mark at the end of a sentence in french. Same thing for the capital, all sentences have them at the start.
@TheoChevalier Is this a thing in france or only in canada?

**Comment by TheoChevalier** [comment]
Yes, we must use non-breaking spaces in this context.
I believe removing them is specific to fr-CA.

You can learn more in our international French styleguide https://github.com/mozfr/besogne/wiki/Guide-stylistique-pour-la-traduction#Rgles_typographiques

**Comment by missuniverse** [comment]
Closing pull request, seems like it was my mistake.

**Pull request closed without merging**